MAiD - Case Management App/force-app/main/default/objects/S1_HC_2023__c/fields/ELG_Q40J__c.field-meta.xml

```diff
@@ -1,11 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
     <fullName>ELG_Q40J__c</fullName>
+    <deprecated>false</deprecated>
```

Comment:
@NataliaNikishina : Looks like this was added, can we remove this, otherwise i am good.

Reply:
Removed